kingpin/actors/aws/elb.py

```diff
@@ -232,3 +197,87 @@ def _execute(self):
         utils.clear_repeating_log(repeating_log)
 
         raise gen.Return()
+
+
+class AddInstance(base.AWSBaseActor):
+
+    """Add an EC2 instance to a load balancer.
+
+    http://boto.readthedocs.org/en/latest/ref/elb.html
+    #boto.ec2.elb.ELBConnection.register_instances
+    """
+
+    all_options = {
+        'elb': (str, REQUIRED, 'Name of the ELB'),
+        'region': (str, REQUIRED, 'AWS region name, like us-west-2'),
+        'instance_id': ((str, list), REQUIRED, 'Instance id, or list of ids')
+    }
+
+    @concurrent.run_on_executor
+    @utils.exception_logger
+    @retry
+    def _add(self, elb, instances):
+        """Invoke elb.register_instances
+
+        This boto function is idempotent, so any retry is OK.
+
+        Args:
+            elb: boto Loadbalancer object
+            instances: list of instance ids.
+        """
+        elb.register_instances(instances)
```

Comment:
Until this is converted into a `gen.coroutine` function it's not easy to catch particular error since everything is `BotoServerError`. However I think putting the `retry` above the `exception_logger` would produce better logs.